### DISCUSSION TITLE
Плохо работает lolminer 1.44 на Windows 3060 rev 1.0 с драйвером 470.05

### DISCUSSION THREAD

**Discussion by OOKKR** [Lolliedieb/lolMiner-releases#1151] [Q&A]
![8777](https://user-images.githubusercontent.com/99263521/152996839-a6c3ccfc-c414-462f-b427-b445752b837c.png)

**Comment by pabloguns** [comment]
> ![8777](https://user-images.githubusercontent.com/99263521/152996839-a6c3ccfc-c414-462f-b427-b445752b837c.png)


the problem could be the "power limit" has to be at 0, and increase the minimum core clock to -400
